ui/src/app/tasks/task-schedule-create/task-schedule-create.component.html

```diff
@@ -0,0 +1,191 @@
+<form class="schedule-form task-create" [formGroup]="form" name="form"
+      *ngIf="schedule$ | async as schedule; else loading" (submit)="submit(schedule)">
+  <div class="heading">
+    <div class="description">
+      <h1>Create schedule(s)</h1>
+      <p>
+        Create a schedule for a task, simply provide a <strong>name</strong> and a <strong>CRON expression</strong>.
+        <br>Optionally, you can add <strong>properties</strong> and <strong>arguments</strong>.
+      </p>
+    </div>
+  </div>
+  <hr>
+  <div>
+    <h2>Setup the schedule</h2>
+    <div *ngIf="schedule.taskDefinitions.length == 1">
+      <div class="row" style="margin-bottom: 12px">
+        <div class="col-xs-4">
+          <label>Task Name:</label>
+        </div>
+        <div class="col-xs-20">
+          <a style="cursor: pointer" (click)="detailsTask(schedule.taskDefinitions[0])">
+            {{ schedule.taskDefinitions[0] }}</a>
+        </div>
+      </div>
+      <div class="row form-group" [class.has-error]="form.get('names').controls[0].invalid">
+        <div class="col-xs-4">
+          <label class="control-label" for="name">Scheduler name:</label>
```

Comment:
Probably want to rename this to Schedule name:

Reply:
+1